libp2p/Common.cpp

```diff
@@ -80,35 +85,50 @@ bool isLocalHostAddress(bi::address const& _addressToCheck)
         {bi::address_v4::from_string(c_localhostIp)},
         {bi::address_v4::from_string("0.0.0.0")},
         {bi::address_v6::from_string("::1")},
-        {bi::address_v6::from_string("::")}
+        {bi::address_v6::from_string("::")},
```

Comment:
Unnecessary comma?

Reply:
This is the way to tell clang-format to list elements in own lines each. Otherwise it will pack it up to the line length limit.

Reply:
wow I didn't know this trick